docker/common/common.dockerfile

```diff
@@ -5,19 +5,19 @@
 # Install dependencies
 ARG APT_DEPENDENCIES=""
 RUN apt-get update && \
-        [ -z "${APT_DEPENDENCIES}" ] || apt-get install --no-install-recommends -y ${APT_DEPENDENCIES} && \
+        [[ -z "${APT_DEPENDENCIES// /}" ]] || apt-get install --no-install-recommends -y ${APT_DEPENDENCIES} && \
```

Comment:
explanation: supports empty spaces, like `APT_DEPENDENCIES=" "` would have failed before. I think it honestly should fail, but added this anyways.

Reply:
The extra spaces do fail for me...

Reply:
![image](https://github.com/uwsbel/autonomy-research-testbed/assets/52421529/895f448f-30e4-4a8c-a4a7-93ad661ebd05)

Reply:
![image](https://github.com/uwsbel/autonomy-research-testbed/assets/52421529/12512c89-2578-4caa-b9ce-0474cf8a9315)
sim container also fails for me, unsure how to fix this one...

Reply:
gotchu, i'll push a fix